Toolchain/BuildIt.sh

```diff
@@ -71,24 +73,24 @@ pushd "$DIR/Build/"
     unset PKG_CONFIG_LIBDIR # Just in case
 
     pushd binutils
-        $DIR/Tarballs/binutils-2.32/configure --prefix=$PREFIX \
-                                              --target=$TARGET \
-                                              --with-sysroot=$SYSROOT \
+        "s"/Tarballs/binutils-2.32/configure --prefix="$PREFIX" \
+                                              --target="$TARGET" \
+                                              --with-sysroot="$SYSROOT" \
                                               --disable-nls || exit 1
-        make -j $MAKEJOBS || exit 1
+        make -j "$MAKEJOBS" || exit 1
         make install || exit 1
     popd
 
     pushd gcc
-        $DIR/Tarballs/gcc-8.3.0/configure --prefix=$PREFIX \
-                                          --target=$TARGET \
-                                          --with-sysroot=$SYSROOT \
+        "s"/Tarballs/gcc-8.3.0/configure --prefix="$PREFIX" \
```

Comment:
This doesn't look right.

Reply:
Not sure how that and the other instance got into the code, I'll fix it right away.